pkg/clone/clone.go

```
@@ -128,6 +134,31 @@ func setUpSshAuth(hostname string) (*ssh2.PublicKeys, error) {
 	return &ssh2.PublicKeys{User: "git", Signer: signer}, nil
 }
 
+func setUpHttpsPersonalAccessToken(hostname, inputToken string) (*string, error) {
+	homeDir, _ := os.UserHomeDir()
```

never swallow any errors. they will lead to ugly panics. although seems like this particular function never fails, it might.

---

done